modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DXPCloudSetup.testcase

```diff
@@ -27,514 +27,667 @@ definition {
-
-		CustomerPortal.accountTableProject(title = "Test Account 01");
-
-		Accounts.addAssigneeToRoleViaUsersTab(
-			accountRoleTitle = "Account Administrator",
-			accountUserName = "Test Test");
+		task ("Creating objects for the DXP Cloud ") {
+			CP2DXPCloudSetup.createDXPCloudObjects();
+		}
 
-		ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
 
-		CustomerPortal.enterOnProject(title = "Test Account 01");
+			CustomerPortal.accountTableProject(title = "Test Account LXC-SM");
 
-		CustomerPortal.clickOnStartProjectSetup();
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
 
-		CustomerPortal.clickOnSkipForNowButton();
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
 
-		CP2DXPCloudSetup.typeAllSetupDXPCloudInfoFields();
+			CustomerPortal.enterOnProject(title = "Test Account LXC-SM");
+		}
 
-		CustomerPortal.clickOnSubmitButton();
+		task ("Going through onboarding to Setup") {
+			CustomerPortal.onboardingFlow(cloudEnvironment = "true");
 
-		AssertElementPresent(locator1 = "CustomerPortalSite#SUCCESS_MESSAGE");
+			Button.click(button = "Submit");
+		}
 
-		CP2DXPCloudSetup.clearDXPCloudInfo();
+		task ("Assert the success message when submitting the form") {
+			AssertElementPresent(locator1 = "CustomerPortalSite#SUCCESS_MESSAGE");
+		}
 
-		CP2DXPCloudSetup.deleteDXPCloudObject();
+		task ("Delete the entries and objects created") {
+			CP2DXPCloudSetup.clearAndDeleteDXPCloudEntries();
+		}
 	}
 
-	@description = "LPS-146684 - Verify if the admin user selects 'Cancel', then the user goes back to the project's Product Activation page"
+	@description = "Automation ID: LPS-146684 | Test Summary: Verify if the admin user selects 'Cancel', then the user goes back to the project's Product Activation page"
 	@priority = "4"
 	test IfSelectsCancelThenTheUserGoesBackPreviousPage {
-		Accounts.openAccountsAdmin();
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
 
-		CustomerPortal.accountTableProject(title = "Test Account 01");
+			CustomerPortal.accountTableProject(title = "Test Account LXC-SM");
 
-		Accounts.addAssigneeToRoleViaUsersTab(
-			accountRoleTitle = "Account Administrator",
-			accountUserName = "Test Test");
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
 
-		ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
 
-		CustomerPortal.enterOnProject(title = "Test Account 01");
+			CustomerPortal.enterOnProject(title = "Test Account LXC-SM");
+		}
 
-		CustomerPortal.clickOnStartProjectSetup();
+		task ("Going through onboardig") {
```

Comment:
Luana, please check the typo: "onboarding"

Reply:
Done.